web/app/js/locales/es/messages.json

```diff
@@ -1,5 +1,103 @@
 {
+  "404Msg": "Página no encontrada.",
   "Add {0} to the k8s.yml file<0/><1/>Then run {inject} to add it to the service mesh": "Agregue {0} a la k8s.yml file<0/><1/>Luego ejecuta {inject} para inyectarla en la malla de servicios",
-  "No data to display": "No hay datos que mostrar",
-  "one or more resources": "uno más recursos más"
+  "All namespaces have a {productName} install.": "Todos los namespaces tienen una instalación de {productName}.",
+  "Control plane components": "Componentes del control plane",
+  "Data plane proxies": "Proxies del data plane",
+  "NoDataToDisplayMsg": "No hay datos que mostrar",
+  "buttonReset": "Resetear",
+  "buttonRunLinkerdCheck": "Ejecuta Linkerd Check",
+  "buttonStart": "Empezar",
+  "buttonStop": "Terminar",
+  "columnTitleAlive": "Vivo",
+  "columnTitleApexService": "Servicio Apex",
+  "columnTitleBest": "Mejor",
+  "columnTitleClusterName": "NombreDelClúster",
+  "columnTitleCount": "Total",
+  "columnTitleDeployment": "Deployment",
+  "columnTitleDirection": "Dirección",
+  "columnTitleEdges": "Bordes",
+  "columnTitleGRPCStatus": "Estado GRPC",
+  "columnTitleGrafana": "Grafana",
+  "columnTitleHTTPStatus": "Estado HTTP",
+  "columnTitleIdentity": "Identidad",
+  "columnTitleJaeger": "Jaeger",
+  "columnTitleLast": "Último",
+  "columnTitleLatency": "Latencia",
+  "columnTitleLeafService": "Servicio Hoja",
+  "columnTitleMeshed": "En la malla de servicios",
+  "columnTitleMeshedPods": "Pods en la malla de servicios",
+  "columnTitleMeshedStatus": "Estado de la malla de servicios",
+  "columnTitleMethod": "Método",
+  "columnTitleName": "Nombre",
+  "columnTitleNamespace": "Espacios de nombres",
```

Comment:
Namespace

Reply:
(I think it's better to leave resource types names untranslated, because that can generate confusion if k8s' UI still returns English labels. I've seen things like "Namespaces" being translated in k8s docs, but then their command examples in the same page still show things in English).